app/views/browse/pod.html

```diff
@@ -142,10 +142,16 @@
 
                       <div class="mar-left-xl mar-bottom-lg">
                         <div class="row">
-                          <div class="pad-left-none pad-bottom-lg col-sm-6 col-lg-4">
+                          <div class="pad-left-none pad-bottom-md col-sm-6 col-lg-4">
```

Comment:
you switched from pad-bottom-lg to pad-bottom-md, is that going to affect the spacing when the dropdown is visible?